src/components/PlayerIcon/PlayerIcon.tsx

```diff
@@ -43,7 +43,7 @@ export function getPlayerIconURL(id, size): Promise<string> {{{
 
 export class PlayerIcon extends React.PureComponent<PlayerIconProps, {url}> {
     mounted: boolean = false;
-    listener;
```

Comment:
sorry not *here* on this line, but in this component, wouldn't let me comment in that method since no changes were there